shared-bindings/index.rst

```diff
@@ -12,13 +12,30 @@ limited. For example, a microcontroller without analog features will not have
 Support Matrix
 ---------------
 
-===============  ==========  =========  ===========  =======  =======  ===========  =================  ===============  ================  =======  =========  ========  =========  ========  =========  =======  =========
-Port             `analogio`  `audioio`  `bitbangio`  `board`  `busio`  `digitalio`  `microcontroller`  `multiterminal`  `neopixel_write`  `os`     `pulseio`  `random`  `storage`  `time`    `touchio`  `uheap`  `usb_hid`
-===============  ==========  =========  ===========  =======  =======  ===========  =================  ===============  ================  =======  =========  ========  =========  ========  =========  =======  =========
-SAMD21           **Yes**     No         No           **Yes**  **Yes**  **Yes**      **Yes**            No               **Yes**           **Yes**  No         **Yes**   **Yes**    **Yes**   **Yes**    Debug    **Yes**
-SAMD21 Express   **Yes**     **Yes**    **Yes**      **Yes**  **Yes**  **Yes**      **Yes**            No               **Yes**           **Yes**  **Yes**    **Yes**   **Yes**    **Yes**   **Yes**    Debug    **Yes**
-ESP8266          **Yes**     No         **Yes**      **Yes**  **Yes**  **Yes**      **Yes**            **Yes**          **Yes**           **Yes**  No         **Yes**   **Yes**    **Yes**   No         Debug    No
-===============  ==========  =========  ===========  =======  =======  ===========  =================  ===============  ================  =======  =========  ========  =========  ========  =========  =======  =========
+=================  =======  ==============  =======
+Module / Port      SAMD21   SAMD21 Express  ESP8266
+=================  =======  ==============  =======
```

Comment:
rst tables are really finicky. I have a feeling the last `======` will need to be longer.

Reply:
I built the docs with sphinx locally to check, and it looks OK. I forgot the ending `==========` line the first time, and pushed a new commit to fix that.
![screenshot from 2017-08-30 13-27-05](https://user-images.githubusercontent.com/2847802/29886111-fbf84a96-8d86-11e7-8187-61fdcc7b7f14.png)